iot_packet_verifier/Makefile

```diff
@@ -0,0 +1,31 @@
+# Makefile for Packet-Verifier service to Helium Packet-Router (HPR)
```

Comment:
We don’t use make files anywhere else here. Why is this needed?

Reply:
Removed as of commit e0e1c6a.  Old habits...